### PR TITLE
setup file for pypi deployment and minor refactoring

### PR DESCRIPTION
**Pull request by Rohan-cod**
## Description

Added `setup.cfg`, `MANIFEST.in`, and `pyproject.toml` for `pypi` deployment and **fixed** some **minor issues**.

## How Has This Been Tested?

The code has been tested by running all the unit tests using the command `python -m unittest` and validated the PEP 8 formatting using `pycodestyle`.

- [x] Unit Tests
- [x] PEP 8 Formatting Validation

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Changes required for PyPI
+[metadata]:
+name = vcf2hl7v2
+version = 0.0.4
```
Comment (srgothi92) [inline setup.cfg]:
```suggestion
version = 0.0.1
```

**Review by srgothi92 — COMMENTED** [review]
Looks good except different types of changes are combined in the same commit.
 Can you help separate refactoring changes into the separate commit from PEP-517 changes.

File: setup.cfg (hunk 1 above)
Comment (Rohan-cod) [reply setup.cfg:4]: Actually, while deploying on test `PyPI`, I faced some errors when installing it via pip due to which I had to deploy 3 more times changing the version every time. The version that is working correctly is [`0.0.4`](https://test.pypi.org/project/vcf2hl7v2/).

**Comment by Rohan-cod** [comment]
> Can you help separate refactoring changes into the separate commit from PEP-517 changes.

Hi @srgothi92, actually I created this pull request from the master branch. Should I close this PR and create two new PRs from different branches or update this PR to reflect only the refactoring changes and send another PR once this is merged?

**Review by srgothi92 — APPROVED** [review]

**Review by rhdolin — APPROVED** [review]

**Pull request merged**